Tests/SwiftStdlibTests/StringExtensionsTests.swift

```diff
@@ -74,10 +74,37 @@ final class StringExtensionsTests: XCTestCase {
         XCTAssertFalse("abc".isAlphaNumeric)
     }
 
-    func testIsEmail() {
-        XCTAssert("john@appleseed.com".isEmail)
-        XCTAssertFalse("john@appleseedcom".isEmail)
-        XCTAssertFalse("johnappleseed.com".isEmail)
+    func testisValidEmail() {
+        // https://blogs.msdn.microsoft.com/testing123/2009/02/06/email-address-test-cases/
```

Comment:
@omaralbeik Maybe we should add the untrimmed email like "     email@mail.com  " .isValidEmail test case as well. What you think?